### DISCUSSION TITLE
Enhance User Control: Add Percentage-Based Swaps to DEX

### DISCUSSION THREAD

**Discussion by 0xjagadbumi** [jumperexchange/jumper-exchange#781] [Ideas]
I have a suggestion for the DEX swap feature: While there's already an option to max out the asset being swapped, it would be incredibly user-friendly and flexible if we could also choose to swap specific percentages, such as 10%, 25%, 50%, or 75%. This would empower users to make more precise and controlled swaps, aligning with diverse trading strategies and risk management preferences.

**Comment by chybisov** [comment]
Thanks for the suggestion, we will see what we can do :)

**Comment by benderofpixels** [comment]
This is a nice idea, especially for those swapping ETH!
